### PR TITLE
modified message when overriding existing quickmark

### LINKED ISSUE #7622: Show current quickmark URL in overwrite dialog
When want to create a new quickmark and a quickmark with that slug already exists, qutebrowser currently asks the user if he wants to overwrite the quickmark. I often wonder what URL is in the current one, it would be awesome if qutebrowser could display the current quickmarks URL in this dialog.

### PR DESCRIPTION
**Pull request by vhadzhiev**
Modified the message when the user wants to create a new quickmark and quickmark with that slug already exists. The new message shows the current marked URL and asks if the user wants to replace it. Closes #7622.

### CODE REVIEW

**Review by mschilli87 — CHANGES_REQUESTED** [review]
nitpick: The commit message has a typo. 😉

**Comment by vhadzhiev** [comment]
> nitpick: The commit message has a typo. 😉

10x, :)

File: qutebrowser/browser/urlmarks.py [hunk 1]
```diff
@@ -186,7 +186,7 @@ def set_mark():
 
         if name in self.marks:
             message.confirm_async(
-                title="Override existing quickmark?",
+                title="Such quickmark already exists for this URL {}.\nOverride existing quickmark?".format(self.marks[name]),
```
Comment (The-Compiler) [inline qutebrowser/browser/urlmarks.py:189]: I feel like the new error message implies that another quickmark already exists for the same URL - but that wouldn't be a problem. What about something like this?

```suggestion
                title=f"Override existing quickmark?\nOld URL: {self.marks[name]}"
```

(also using [f-strings](https://fstring.help/) for more modern string formatting - we don't use it consistently yet, but it should be preferred for new code)

**Review by The-Compiler — CHANGES_REQUESTED** [review]
Thanks! I added a suggestion on a better message wording.

The failing tests should be fine, looks like we have a couple of new flakiness issues with the recent Qt 6 merge.

File: qutebrowser/browser/urlmarks.py (hunk 1 above)
Comment (vhadzhiev) [reply qutebrowser/browser/urlmarks.py:189]: I initially did it with f-string, but then saw the formatting in the file and decided to make it the same for consistency.
If you think that useful I can start refactoring the files to f-string in my free time.

Comment (vhadzhiev) [reply qutebrowser/browser/urlmarks.py:189]: Did I cancel the your suggestion by mistake when I corrected the typo in my commit? I did this  https://github.com/qutebrowser/qutebrowser/pull/7635#issuecomment-1479147811 in order to correct the typo in the commit message.

**Comment by The-Compiler** [comment]
Looks like there was a syntax error in my suggestion - sorry! Could you please fix that up?

> If you think that useful I can start refactoring the files to f-string in my free time.

Thanks for the offer - but there are tools like [pyupgrade](https://github.com/asottile/pyupgrade) and [flynt](https://github.com/ikamensh/flynt) which can automate that work. The main issue right now is that doing so introduces a lot of code churn, causing more conflicts which currently open PRs. But for 4.0.0 we plan to take that step: #1455.

**Comment by vhadzhiev** [comment]
> Looks like there was a syntax error in my suggestion - sorry! Could you please fix that up?
> 
> > If you think that useful I can start refactoring the files to f-string in my free time.
> 
> Thanks for the offer - but there are tools like [pyupgrade](https://github.com/asottile/pyupgrade) and [flynt](https://github.com/ikamensh/flynt) which can automate that work. The main issue right now is that doing so introduces a lot of code churn, causing more conflicts which currently open PRs. But for 4.0.0 we plan to take that step: #1455.

I can do it, but this will probably create new commit. And I'm not very sure how to deal with the squashing.

> LGTM now, though I'd prefer squashing the second commit into the first one for a cleaner Git log.

Maybe you can squash everything after that?

**Comment by The-Compiler** [comment]
Don't worry about it - I can indeed just squash-merge this.

**Comment by mschilli87** [comment]
@vhadzhiev
> Maybe you can squash everything after that?

I won't, but you and @The-Compiler will.

If you have a local checkout of the branch, you can run `git rebase --interactive 787c417fbd7721908d26618e7b1fd9dddacdae5f` which will open your editor prompting you for what to do with the additional commits. You can modify the corresponding lines to start with `s` for squashing or `f` to mark them as a fixup (which will drop their commit message after squashing), save and exit the editor and update the PR via `git push --force` once the rebase was done (you can verify it worked as intented via `git log`).

File: qutebrowser/browser/urlmarks.py [hunk 2]
```diff
@@ -186,7 +186,7 @@ def set_mark():
 
         if name in self.marks:
             message.confirm_async(
-                title="Override existing quickmark?",
+                title=f"Override existing quickmark?\nOld URL: {self.marks[name]}",
```
Comment (arza-zara) [inline qutebrowser/browser/urlmarks.py:189]: The `\n` doesn't work in the title, and anyway it looks a bit better and is wrappable as body text
longoverride1
longoverride2

```suggestion
                title=f"Override existing quickmark?",
                text=f"Old URL: {self.marks[name]}",

```

**Review by arza-zara — CHANGES_REQUESTED** [review]

**Pull request closed without merging**